resources/python-examples.md

```diff
@@ -0,0 +1,30 @@
+<p align="center">
+
+## Contents
+- [Documentation](#documentation-links)
+- [Code Examples](#code-examples)
+
+
+## Documentation Links
+- Neo Python: [GitHub](https://github.com/CityOfZion/neo-python/) | [Documentation](https://neo-python.readthedocs.io/en/latest/)
+- Neo Boa: [GitHub](https://github.com/CityOfZion/neo-boa) | [Documentation](http://neo-boa.readthedocs.io/en/latest/)
+
+
+## Code Examples
+
+### Smart Contract Examples
+- [Neo Python smart contract](https://github.com/CityOfZion/neo-python/blob/master/examples/smart-contract.py) - An example of a Neo Python Smart contract, provided by Neo Python.
+- [Splyse Neo Smart IoT smart contract](https://github.com/Splyse/neo-smart-iot/blob/master/elcaro-contract.py) - DApp Competition entrant.
+- [BlockAuth smart contract](https://github.com/CityOfZion/neo-python/blob/master/examples/smart-contract.py) - DApp Competition entrant.
+- [LuckyNeo smart contract](https://github.com/mmoravec/luckyneo/blob/master/LuckyNeo.py) - DApp Competition entrant.
+- [Krpyton smart contract](https://github.com/MediaServe/KRYPTON/blob/master/contract/krypton.py) - DApp Competition entrant.
+- [Sunny DApp smart contract](https://github.com/JorritvandenBerg/sunny-dapp/blob/master/smartcontract/sunny_dapp.py) - DApp Competition entrant.
+
+### Middleware Examples
+- [Splyse Neo Smart IoT middleware](https://github.com/Splyse/neo-smart-iot/blob/master/neo-pubsub.py) - DApp Competition Entrant. IOT device middleware.
```

Comment:
Keep the capitalisation of 'entrant' consistent; either make it capitalised everywhere or nowhere